java-client/src/main/java/org/opensearch/client/opensearch/core/msearch/MultisearchBody.java

```diff
@@ -364,6 +383,8 @@ public static class Builder extends ObjectBuilderBase implements ObjectBuilder<M
 		@Nullable
 		private SourceConfig source;
 
+		private Map<String, ScriptField> scriptFields;
```

Comment:
It can be empty but if you explicitly pass `null` to the Builder, it fails for `mapPutAll`.  Also, the server does not return the right results when the body contains `null`.